.circleci/config.yml

```diff
@@ -67,13 +68,34 @@ jobs:
         environment:
           GOGC: "20"
           GOOPTS: "-p 2 -mod=vendor"
+
+  test_mixins:
+    docker:
+    - image: quay.io/coreos/jsonnet-ci:release-0.37
```

Comment:
This image will be deprecated soon as it doesn't have a purpose anymore. It was needed before to easily ship C++ version of `jsonnetfmt` but this binary is included in go version of jsonnet v0.16.

I recommend using separate `go.mod` file and installing tooling dependencies from there or adding `jsonnet` to circle orb.

Reply:
Correction. This image is already deprecated.

Reply:
:+1: